src/bundle/ui-dev/src/modules/common/alert/alert.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Icon from '../icon/icon';
+import { createCssClassNames } from '../helpers/css.class.names';
+
+const ICON_NAME_MAP = {
+    info: 'system-information',
+    error: 'circle-close',
+    warning: 'warning-triangle',
+    success: 'checkbox',
+};
+
+const Alert = ({
+    type,
+    title,
+    subtitle,
+    iconName: iconNameProp,
+    iconPath,
+    showSubtitleBelow,
+    showCloseBtn,
+    onClose,
+    extraClasses,
+    children,
+}) => {
+    const className = createCssClassNames({
```

Comment:
do we want to use this helper if all classes are always visible?

Reply:
Personally, this way code is more clear. For you, one-line concatenation is clearer?